### ISSUE TITLE
Announcing OSHI-FFM!   Contributors wanted!

### ISSUE THREAD

**Issue by dbwiddis** [oshi/oshi#2191]
TLDR: Click here: https://github.com/oshi/oshi-ffm/issues/4 

Today marked the GA release of JDK 19.  I've been looking forward to this release for almost a year.

The big news for OSHI users is the Preview Release of Java's **Foreign Function & Memory API**, [JEP 424](https://openjdk.org/jeps/424).  This is part of Project Panama, enabling native function invocation and memory allocation from within Java, without the overhead of JNA (reflection + FFI).  The Preview phase means a mostly-stable API, and much easier integration into IDEs with only a few command line arguments needed to enable the features.

Why is this a big deal?   Some [benchmarks](https://github.com/zakgof/java-native-benchmark#results) indicate a potential 12-13x improvement.  My actual experience so far [informally benchmarking OSHI](https://github.com/oshi/oshi-ffm/issues/5) shows about a 5x improvement.

So while OSHI will continue on in its current form for the near future, OSHI-FFM will enable an order of magnitude performance improvement.

The goal of the new project is to port over all the existing OSHI capability (with possible minor API updates) over the next year, targeting a 1.0.0 release about the same time as JDK 21 LTS in September 2023.

Here's the good news for prospective contributors:
 - There is a shiny new repository waiting for you to join and contribute here:  https://github.com/oshi/oshi-ffm
 - Want to learn about the new Foreign Function and Memory API? You are welcome to use this new project as a practical training ground! What better way to sharpen your skills than to solve an actual, practical problem instead of repeating a Hello World tutorial!  Get in early before all the easy methods are taken! ;)
 - Want to have a voice in the future of OSHI? Contribute more and join the maintainer team.  This project requires a community effort and community maintainership.  I'd personally like to retire as OSHI's benevolent dictator. :)

If you found this issue based on the [Hacktoberfest](https://hacktoberfest.com/) tag, head on over to https://github.com/oshi/oshi-ffm/issues/4 where you will have plenty of options to amass PRs to get your swag.

**Comment by dbwiddis** [comment]
Closing this issue due to lack of interest.  Planning a different approach for integrating Panama features.
